package.json

```diff
@@ -23,9 +23,12 @@
     "async": "^0.9.0",
     "chalk": "^1.0.0",
     "cli-table": "^0.3.1",
-    "commander": "^2.7.1",
+    "commander": "^2.9.0",
```

Comment:
Really, no, but since 2.8.0 commander.js will display help if you run root command without subcommand, which is a nice thing. Also, if we decide to have a default command instead of displaying help, 2.9.0 added that.

https://github.com/tj/commander.js/blob/master/History.md